libdevcore/Terminal.h

```diff
@@ -7,72 +7,72 @@ namespace con
 
 #ifdef _WIN32
 
-#define EthReset ""       // Text Reset
-
-#define EthReset ""       // Text Reset
-
-	// Regular Colors
-#define EthBlack	""    // Black
-#define EthCoal		""    // Black
-#define EthGray		""    // White
-#define EthWhite	""    // White
-#define EthMaroon	""    // Red
-#define EthRed		""    // Red
-#define EthGreen	""    // Green
-#define EthLime		""    // Green
-#define EthOrange	""    // Yellow
-#define EthYellow	""    // Yellow
-#define EthNavy		""    // Blue
-#define EthBlue		""    // Blue
-#define EthViolet	""    // Purple
-#define EthPurple	""    // Purple
-#define EthTeal		""    // Cyan
-#define EthCyan		""    // Cyan
-
-#define EthBlackBold	""      // Black
-#define EthCoalBold		""      // Black
-#define EthGrayBold		""      // White
-#define EthWhiteBold	""      // White
-#define EthMaroonBold	""      // Red
-#define EthRedBold		""      // Red
-#define EthGreenBold	""      // Green
-#define EthLimeBold		""      // Green
-#define EthOrangeBold	""      // Yellow
-#define EthYellowBold	""      // Yellow
-#define EthNavyBold		""      // Blue
-#define EthBlueBold		""      // Blue
-#define EthVioletBold	""      // Purple
-#define EthPurpleBold	""      // Purple
-#define EthTealBold		""      // Cyan
-#define EthCyanBold		""      // Cyan
-
-	// Background
-#define EthOnBlack		""       // Black
-#define EthOnCoal		""		 // Black
-#define EthOnGray		""       // White
-#define EthOnWhite		""		 // White
-#define EthOnMaroon		""       // Red
-#define EthOnRed		""       // Red
-#define EthOnGreen		""       // Green
-#define EthOnLime		""		 // Green
-#define EthOnOrange		""       // Yellow
-#define EthOnYellow		""		 // Yellow
-#define EthOnNavy		""       // Blue
-#define EthOnBlue		""		 // Blue
-#define EthOnViolet		""       // Purple
-#define EthOnPurple		""		 // Purple
-#define EthOnTeal		""       // Cyan
-#define EthOnCyan		""		 // Cyan
-
-	// Underline
-#define EthBlackUnder	""       // Black
-#define EthGrayUnder	""       // White
-#define EthMaroonUnder	""       // Red
-#define EthGreenUnder	""       // Green
-#define EthOrangeUnder	""       // Yellow
-#define EthNavyUnder	""       // Blue
-#define EthVioletUnder	""       // Purple
-#define EthTealUnder	""       // Cyan
+void HandleLogOutput(std::string const& _s);
+
+#define EthReset "\x1b[0m"       // Text Reset
+
+// Regular Colors
+#define EthBlack "\x1b[30m"        // Black
```

Comment:
Actually I just copied the Unix defines to Windows defines - I believe ANSI codes are the same on all systems.